hash/ibm5170.xml

```diff
@@ -12816,6 +12984,32 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="wcgpsa" cloneof="f1gp">
```

Comment:
Wouldn't it be better to give the later v1.04 the shorter set name "wcgps" and the earlier v1.03 the alternate set name "wcgpsa"?

Reply:
Done. Yes, indeed it's better to switch the names.